### ISSUE TITLE
Remove `languageId` from markdown cells

### ISSUE THREAD

**Issue by christian-bromann** [stateful/runme#91]
It seems that for markdown sections we return a `languageId` property that is always empty. I guess we can remove this?

**Comment by adambabik** [comment]
I was thinking about hiding it when it's empty but if I read TS correctly, `NotebookCellData` [requires presence of `languageId` always](https://github.com/microsoft/vscode/blob/085c409898bbc89c83409f6a394e73130b932add/src/vscode-dts/vscode.d.ts#L13731).

**Comment by christian-bromann** [comment]
That is true, however it should not be a concern for the runme CLI as it should provide an ideal interface for what it is doing and not be too dependent on VS Code. In fact we don't even use/access the property. So there is not really a value to be 1:1 compliant with the VS Code API.
